### PR TITLE
updating getting-started setup instructions

### PR DESCRIPTION
**Pull request by alessandrozucca**

### CODE REVIEW

File: content/remote-development/getting-started.md [hunk 1]
```diff
@@ -36,33 +36,31 @@ brew install continuouspipe/tools/cp-remote
...
 ### Windows (64-bit):
 
-* Download https://continuouspipe.github.io/remote-environment-client/0.1.0/windows-amd64.gz
-* Extract `cp-remote.gz`
+* Download https://continuouspipe.github.io/remote-environment-client/latest/windows-amd64/cp-remote.zip
+* Extract `cp-remote.zip`
 * Move `cp-remote.exe` into your project folder
 
-If your architecture is 32-bit use `windows-386.gz` rather than `windows-amd64.gz`.
+If your architecture is 32-bit use `latest/windows-386/` rather than `latest/windows-amd64/`.
```
Comment (tonypiper) [inline content/remote-development/getting-started.md]: perhaps best to list the two full URLs in case it's confused for a folder inside the zip file.

Comment (rightsaidjames) [inline content/remote-development/getting-started.md]: Or perhaps just the full 32-bit URL, e.g:

> If your architecture is 32-bit download https://continuouspipe.github.io/remote-environment-client/latest/windows-386/cp-remote.zip instead.

Comment (alessandrozucca) [inline content/remote-development/getting-started.md]: should we have 2 separate sections?

Comment (alessandrozucca) [inline content/remote-development/getting-started.md]: Linux (64-bit):
Linux (32-bit):
Windows (64-bit):
Windows (32-bit):

**Comment by tonypiper** [comment]
@rightsaidjames any better? 

http://35.186.193.108/remote-development/getting-started/

File: content/remote-development/getting-started.md [hunk 2]
```diff
@@ -36,33 +36,40 @@ brew install continuouspipe/tools/cp-remote
 Otherwise you can install it manually via:
 
 ```
-sudo curl https://continuouspipe.github.io/remote-environment-client/0.1.0/darwin-amd64.gz > cp-remote.gz
-gzip -d cp-remote.gz;
+sudo curl https://continuouspipe.github.io/remote-environment-client/latest/darwin-amd64/cp-remote.tar.gz > cp-remote.tar.gz
+tar -xzvf cp-remote.tar.gz;
```
Comment (pswaine) [inline content/remote-development/getting-started.md]: I think when you are bundling tar options, the best practice is to drop the preceding dash.

Comment (alessandrozucca) [inline content/remote-development/getting-started.md]: Ah that's the "Old Option Style" - which i never heard of until today 😄 
http://www.gnu.org/software/tar/manual/tar.html#subsection

so having the dash is okay
-x extract
-z gzip
-v verbose
-f file target

Comment (pswaine) [inline content/remote-development/getting-started.md]: Now I come to think of it - I received that advice 10 years ago :)

Comment (alessandrozucca) [inline content/remote-development/getting-started.md]: 😆

**Comment by rightsaidjames** [comment]
So the download and extract instructions worked, but if I move cp-remote.exe into my project folder I just get:
```jamesthomas@JAMES-PC MINGW64 ~/dev/cloudinary-magento2 (master)
$ cp-remote
bash: cp-remote: command not found```

However once I put it in a folder that was in my PATH it worked as expected:
```jamesthomas@JAMES-PC MINGW64 ~/dev/cloudinary-magento2 (master)
$ cp-remote version
Current version: 0.1.0 (windows-amd64)
```

**Comment by sroze** [comment]
@rightsaidjames good point for Windows!

**Comment by rightsaidjames** [comment]
Turns out that if you use the normal command line or PowerShell then putting it in the project folder works fine, it's only in Git Bash that it doesn't work. And since I couldn't get `watch` or `bash` to work in Git Bash anyway, I think it's fine to leave it as-is.

**Comment by rightsaidjames** [comment]
Additional notes from testing:
- To get `watch` to work properly, I had to ensure that the contents of cwRsync's bin folder were also in my path. Before I did this, file syncing appeared to work but didn't actually do anything beyond report that files had been synced.
- Trying to use `watch` or 1bash` in Git Bash gives me:
 `Unable to use a TTY - input is not a terminal or the right kind of file`

**Comment by alessandrozucca** [comment]
Hi @rightsaidjames, can I merge this PR ?

I guess the main thing for this PR is that the links are working. 
If there are things that needs to be investigate for `rsync` on Windows we can sort them on a different remote client ticket

**Comment by rightsaidjames** [comment]
@alessandrozucca Yup, merge away (sorry, didn't realise you were waiting on me). I think the instructions are fine - I got the cp-remote installed fine and I'm sure we could put some extra instructions for `rsync` somewhere else in the future.

**Comment by alessandrozucca** [comment]
Thank you @rightsaidjames 😄

**Pull request merged**